.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - run: make ui-lint
       - run: make ui-build-module
       - run: make ui-test
-      - run: make assets-tarball -o assets
+      - run: make assets-tarball
```

Comment:
in order to save a bit of action and time you could simplify this list by : 

```yaml
- run: make assets-tarball
- run: make ui-lint
- run: make ui-test
```

Reply:
done